src/printer.js

```diff
@@ -114,13 +114,13 @@ function genericPrint(path, options, printPath, args) {
   }
 
   if (needsParens) {
-    parts.unshift("( ");
+    parts.unshift("(", options.calypsoSpacing ? " " : "");
```

Comment:
> Wouldn't it be simplest if at the top of the file you had:
> `const calypsoSpace = options.calypsoSpacing ? " " : ""`

`options` is a parameter to `genericPrint` and `genericPrintNoParens`, so these definitions can't be at top of the file. I'll repeat the definition of `calypsoSpace` at the beginning of both print functions.